src/services/completions.ts

```diff
@@ -981,6 +981,7 @@ namespace ts.Completions {
                 completionNodes.push(node);
             },
             body,
+            /* overridesAbstract */ false,

```

Comment:
I don't think that should always be false, otherwise the completion will suffer from the same bug we're trying to fix here for the codefix.

Reply:
Instead, we should check if the method we're possibly overriding is abstract.